CMake/unix_config.cmake

```diff
@@ -21,7 +21,7 @@ macro(os_set_flags)
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
         set(LRS_TRY_USE_AVX true)
-    endif(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    endif(${MACHINE} MATCHES "arm-.*linux.*")
```

Comment:
@raymanfx hello,
The current development/ac branches have a similar ( if not more aggressive) relaxation.
Can you check and update if it can work for you?